test/jdk/sun/security/tools/jarsigner/compatibility/Compatibility.java

```diff
@@ -1439,7 +1439,9 @@ private SignItem tsaDigestAlgorithm(String tsaDigestAlgorithm) {
         }
 
         String expectedTsaDigestAlg() {
-            return tsaDigestAlgorithm != null ? tsaDigestAlgorithm : "SHA-256";
```

Comment:
You can actually get it from `jdk.security.jarsigner.JarSigner.Builder.getDefaultDigestAlgorithm()`.

Reply:
I updated the code to use `getDefaultDigestAlgorithm` but I still need the version-check because this test invokes jarsigner in different JDKs that use different default algorithms.